app/pages/control.py

```diff
@@ -0,0 +1,358 @@
+                                DashIconify(icon="solar:upload-outline", width=100)
+                            ],
+                        ),
+                        html.Div(
+                            id="button_undo",
+                            style={
+                                "backgroundColor": "gray",
+                                "border-radius": "50%",
+                                "padding": "20px",
+                                "display": "grid",
+                                "justify-content": "center",
+                            },
+                            children=[DashIconify(icon="iconoir:undo", width=100)],
+                        ),
+                    ],
+                ),
+                html.Div(
+                    style={
+                        "display": "flex",
+                        "justify-content": "space-around",
+                        "padding": "10px",
+                    },
+                    children=[
+                        html.Div(
+                            id="button_start",
+                            style={
+                                "backgroundColor": "gray",
+                                "border-radius": "50%",
+                                "padding": "20px",
+                                "display": "grid",
+                                "justify-content": "center",
+                            },
+                            children=[DashIconify(icon="ph:play-fill", width=100)],
+                        ),
+                        html.Div(
+                            id="button_stop",
+                            style={
+                                "backgroundColor": "gray",
+                                "border-radius": "50%",
+                                "padding": "20px",
+                                "display": "grid",
+                                "justify-content": "center",
+                            },
+                            children=[DashIconify(icon="ri:stop-fill", width=100)],
+                        ),
+                        html.Div(
+                            id="button_refresh",
+                            style={
+                                "backgroundColor": "gray",
+                                "border-radius": "50%",
+                                "padding": "20px",
+                                "display": "grid",
+                                "justify-content": "center",
+                            },
+                            children=[
+                                DashIconify(icon="solar:refresh-bold", width=100)
+                            ],
+                        ),
+                    ],
+                ),
+            ],
+        ),
+    ],
+)
+
+
+@callback(Output("button_save", "className"), [Input("button_save", "n_clicks")])
+def save_button_click(n_clicks):  # type: ignore # noqa
+    """Placeholder function for Save Button."""
```

Comment:
I think I forgot to mention that the [Whimsical board](https://whimsical.com/idles-LP5cXQpyXNxtqA8fr6BKA9) contains the details of what these buttons should do. Can you change the function names and docstrings for these to more closely match those? Also, can you include more inputs/function parameters you think these callbacks might need?

(Also, I don't think we need the [square brackets] around `Input` anymore)

For example, this one could be:

```suggestion
@callback(
    Output("button_save", "children"),
    Input("button_save", "n_clicks"),
    State("Hub01_dropdown", "value"),
    etc...
)
def update_displays_button_click(n_clicks, hub01):  # type: ignore # noqa
    """Placeholder function for the Update Displays Button."""
```